subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/DefaultSingletonFileTreeTest.groovy

```diff
@@ -45,4 +50,31 @@ class DefaultSingletonFileTreeTest extends Specification {
         }
         0 * visitor._
     }
+    
+    def "can be converted to a directory tree"() {
```

Comment:
Should we mention the `@Issue` that triggered this?

Reply:
Good point. Added.